packages/cli/src/commands/nuke.ts

```diff
@@ -54,6 +55,9 @@ export default class Nuke extends Command {
 
   public async run(): Promise<void> {
     const { flags } = this.parse(Nuke)
+
+    await checkCurrentDirBoosterVersion(this.config.userAgent)
```

Comment:
I think we're calling this on every CLI command right? Is there a way that it could be called before all of them?

Reply:
Yep I was thinking in the same thing. Perhaps with the [Custom Base Class feature from `oclif`](https://oclif.io/docs/base_class)?

Reply:
It's a good idea to inherit from a BaseCommand class. Actually, this refactoring can help us to address other issues from the list